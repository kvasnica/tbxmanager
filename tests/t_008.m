function t_008
% tests double installing

onCleanup(@(x) tbx_setupTest('done')); tbx_setupTest('start');

% set up the test repo
tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml

tbxmanager install tbx1
tbxmanager install tbx1

end
