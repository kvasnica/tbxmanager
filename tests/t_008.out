--- tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml 
--- tbxmanager install tbx1 

Installing version "1.0" of "tbx1"...
Downloading "http://control.ee.ethz.ch/~mpt/tbx/test/tbx1_v1.zip"...
"tbx1:1.0:all" installed to /Users/michal/matlab/mpt3/hg/tbxmgr-mk/tests/tbxstorage_test/tbx1/1.0/all
Toolbox "tbx1:1.0:all" added to the Matlab path.
--- tbxmanager install tbx1 
Latest version of "tbx1" is already installed.
